sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -936,6 +941,16 @@ public enum BlobType
         Page = 1,
         Append = 2,
     }
+    public partial class BlobUploadFromUriOptions
```

Comment:
this should take metadata as well.

Reply:
Unfortunately there is a service bug, setting metadata on doesn't currently work.

https://github.com/Azure/azure-sdk-for-net/issues/15969